lib/FieldType/Mapper/RelationListFormMapper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\Core\FieldType\RelationList\Type;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+
+class RelationListFormMapper implements FieldTypeFormMapperInterface
+{
+    /**
+     * @var Repository The API Repository, used to fetch list of available content types
+     */
+    protected $repository;
+
+    /**
+     * @var ConfigResolverInterface Config resolver, for reading current language
+     */
+    protected $configResolver;
+
+    /**
+     * @param Repository $repository
+     * @param ConfigResolverInterface $configResolver
+     */
+    public function __construct(Repository $repository, ConfigResolverInterface $configResolver)
+    {
+        $this->repository = $repository;
```

Comment:
You don't need the whole repository here, only the ContentService. Please only inject it.